docs/sdks/js-reference-guide/js-sdk-redeemOtp.md

```diff
@@ -0,0 +1,69 @@
+---
+title: redeemOtp
+id: js-reference-redeemOtp
+description: ''
+slug: /js-reference-redeemOtp
+keywords:
+ - javascript sdk
+pagination_next: null
+pagination_prev: null
+last_update:
+   date: 07/25/2023
+   author: William May
+draft: false
+doc_type: reference
+displayed_sidebar: sdkSidebar
+---
+
+The **redeemOtp** function enables an app using the Beyond Identity Javascript SDK to redeem an otp for a grant code.
```

Comment:
might be helpful to mention that code was the result to authenticateOtp

Reply:
Added.